platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/events/performance/IThreadMetricsProvider.kt

```diff
@@ -0,0 +1,18 @@
+// capture-sdk - bitdrift's client SDK
+// Copyright Bitdrift, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+package io.bitdrift.capture.events.performance
+
+/**
+ * Provides attributes for threads running in the app
+ */
+interface IThreadMetricsProvider {
+    /**
+     * Returns Thread information (e.g. Thread count)
+     */
+    fun getThreadAttributes(): Map<String, String>
```

Comment:
As BIT-371 say could you add this to resource utilization logs as well? without it this information can only be available for users that have crashed vs every single user